layouts/shortcodes/video-item.html

```diff
@@ -0,0 +1,57 @@
+{{ $inner := .Get "body" }}
+{{ $law := .Get "law" }}
+{{ $width := .Get "width" | int }}
+{{ $height := .Get "height" | int }}
+{{ $pb := slice $width $height | partial "helper/aspect-ratio" }}
+{{ $caption := .Get "caption" | default "No caption" }}
+
+
+<figure
+  class="mx-auto md:max-w-5xl"
+  x-data="{ onScreen: false, splPlayer: null }"
+  x-init="
+    splPlayer = new Vimeo.Player(
+      $refs.splVideo, {
+        id: $refs.splVideo.dataset.dataVimeoId,
```

Comment:
@carlmjohnson please take a look at this when you can. I cannot seem to get Alpine to grab hold of the vimeo id to make it work.